financial-services-accelerator/accelerators/fs-apim/carbon-home/repository/resources/conf/templates/repository/conf/financial-services.xml.j2

```diff
@@ -25,7 +25,7 @@
         <RequestRouter>org.wso2.financial.services.accelerator.gateway.executor.core.DefaultRequestRouter</RequestRouter>
         {% endif %}
         <FinancialServicesGatewayExecutors>
-            {% for type in financial_services.gateway.openbanking_gateway_executors.type %}
+            {% for type in financial_services.gateway.gateway_executors.type %}
```

Comment:
{% for type in financial_services.`gateway.gateway`_executors.type %}

Do we need gateway twice here?

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/130/commits/1e13ec5af065388311c13d8ca7f04ef804d84184